### ISSUE TITLE
No 'Access-Control-Allow-Origin'

### ISSUE THREAD

**Issue by jacklp** [etherdelta/etherdelta.github.io#191]
Trying to make a simple request via a react app:

```
request({
          url: 'https://api.etherdelta.com/returnTicker',
          method: 'GET',
          json: true
        }, (error, response, body) => {
          console.log(body, error, response);
        })
```

I get the following error: 

> Failed to load https://api.etherdelta.com/returnTicker: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:8000' is therefore not allowed access. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

**Comment by zackcoburn** [comment]
Yes, the API has CORS enabled. So if you want your site to be on the CORS
whitelist, you'll have to ask for permission. Go to https://zackcoburn.com
and use the button at the bottom to email me. Happy to whitelist
non-malicious apps. Cheers!

Zack Coburn

NOTICE: This e-mail transmission (and/or the attachments accompanying it)
may contain confidential information belonging to the sender which is
protected by law. The information is intended only for the use of the
intended recipient. If you are not the intended recipient, you are hereby
notified that any disclosure, copying, distribution or the taking of any
action in reliance on the contents of this information is strictly
prohibited. Any unauthorized interception of this transmission is illegal.
If you have received this transmission in error, please promptly notify the
sender by reply e-mail, and then destroy all copies of the transmission.
